_episodes/08-vector-open-shapefile.md

```diff
@@ -21,41 +21,36 @@ keypoints:
 
 Starting with this episode, we will be moving from working with raster
 data to working with vector data. In this episode, we will open and plot point, line and polygon vector data
-stored in shapefile format in Python. These data refer to the [NEON Harvard Forest field site](https://www.neonscience.org/field-sites/field-sites-map/HARV), which we have been working with in previous
-episodes. In later episodes, we will learn how to work with raster and
-vector data together and combine them into a single plot.
+stored in shapefile format in Python.
 
-## Import Shapefiles
+The data we will use comes from the Dutch government's open geodata sets obtained from the [PDOK platform](https://www.pdok.nl/).
```

Comment:
```suggestion
The data we will use comes from the Dutch government's open geodata sets obtained from the [PDOK platform](https://www.pdok.nl/). It provides open data for various applications, e.g. cadaster, infrastructure, agriculture, etc. In this episode we will use three data sets:

- [Crop fields](https://www.pdok.nl/introductie/-/article/basisregistratie-gewaspercelen-brp-) (polygons)
- [Water ways](https://www.pdok.nl/introductie/-/article/nationaal-wegen-bestand-nwb-) (lines):
- [Groud water monitoring wells](https://www.pdok.nl/downloads/-/article/basisregistratie-ondergrond-bro-) (points): 
```

Reply:
I add a suggestion for the data introduction part. Here I listed three datasets following our discussion in issue #4 and the later intro/excercises.

Reply:
Yes, I like the suggested text. I've added it now.